frontend/src/common/config.ts

```diff
@@ -14,12 +14,18 @@ const axios = AxiosStatic.create({
   }
 });
 
-export class ConfigDto {
-  constructor(public keycloakRealm: string, public keycloakUrl: string, public keycloakClientId: string, public keycloakAuthEndpoint: string, public keycloakTokenEndpoint: string) { }
-}
+export type ConfigDto = {
+  keycloakRealm: string;
+  keycloakUrl: string;
+  keycloakClientId: string;
+  keycloakAuthEndpoint: string;
+  keycloakTokenEndpoint: string;
+  serverTime: string;
+};
 
 class ConfigWrapper {
   private data: ConfigDto;
+  private serverTimeDiff: number;
```

Comment:
instead of a private property and a public getter, use:

```suggestion
  readonly serverTimeDiff: number;
```

Reply:
Implemented in https://github.com/cryptomator/hub/pull/164/commits/96a104ec834fc75db9695a822462c5ca6c0e2f0f and https://github.com/cryptomator/hub/pull/164/commits/e373fb727a461e684eae89efcd068326e0d67611